client/package.json

```diff
@@ -54,6 +54,7 @@
     "markdown-it": "^11.0.0",
     "markdown-it-regexp": "^0.4.0",
     "moment": "2.26.0",
+    "moment-duration-format": "^2.3.2",
```

Comment:
Is this firmly locked to moment?  I was actually hoping to dump moment soon for a more modern lighter weight alternative soon.

Reply:
This is used in `moment.duration(diff).format("hh [hours], mm [minutes], ss [seconds]")` which is nice since it skips 0 values from the left. I did switch this to luxon and iso format strings ... I guess that's just as readable and more standard.

Reply:
So ... date-fns does nicer formatting, and it also seems to be smaller and faster, and it has the big bonus of not breaking the jest tests 😆.